From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Thu, 9 Apr 2015 17:24:40 +0200
Subject: [PATCH] build-sys: Enable large file support

This is needed on 32 bit machines for drag and drop of large files (>2GB
to work). This fixes https://bugzilla.redhat.com/show_bug.cgi?id=1206663
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 377fc74..9f9d34c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -10,6 +10,7 @@ m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_HEADER_STDC
+AC_SYS_LARGEFILE
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_DEFINE(_GNU_SOURCE, [1], [Enable GNU extensions])
